src/TSMapEditor/Models/AITriggerType.cs

```diff
@@ -91,6 +91,7 @@ public AITriggerType(string iniName)
         public bool Easy { get; set; } = true;
         public bool Medium { get; set; } = true;
         public bool Hard { get; set; } = true;
+        public bool Enabled { get; set; } = false;
```

Comment:
We surely want AI triggers to be enabled by default.

```suggestion
        public bool Enabled { get; set; } = true;
```